src/main/java/net/snowflake/client/core/HttpUtil.java

```diff
@@ -295,9 +307,18 @@ public static CloseableHttpClient buildHttpClient(
       // only set the proxy settings if they are not null
       // but no value has been specified for nonProxyHosts
       // the route planner will determine whether to use a proxy based on nonProxyHosts value.
+      StringBuilder logBuilder = new StringBuilder();
+      logBuilder.append(String.format("Rebuilding request config. Connect timeout: %d ms, connection request timeout: %d ms, " +
```

Comment:
But if you really want to optimize, string builder would be better than string formatting/replace ;)

Reply:
I agree - we should use Builder or formatter - it's not optimization to use them both

Reply:
Right, changed.